src/main/kotlin/dev/arbjerg/ukulele/config/BotProps.kt

```diff
@@ -9,5 +9,6 @@ class BotProps(
         var prefix: String = "::",
         var database: String = "./database",
         var game: String = "",
-        var trackDurationLimit: Int = 0
+        var trackDurationLimit: Int = 0,
+        var idleTimeMinutes: Int = 10
```

Comment:
I think this should be disabled by default
```suggestion
        var idleTimeMinutes: Int = 0
```